src/test/java/com/blackducksoftware/integration/hub/alert/datasource/entity/distribution/SlackConfigEntityTest.java

```diff
@@ -14,14 +14,15 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
-import com.blackducksoftware.integration.hub.alert.datasource.entity.distribution.SlackDistributionConfigEntity;
+import com.blackducksoftware.integration.hub.alert.mock.MockUtils;
 import com.blackducksoftware.integration.hub.alert.mock.SlackMockUtils;
 
 public class SlackConfigEntityTest extends EntityTest<SlackDistributionConfigEntity> {
-    private static final SlackMockUtils mockUtils = new SlackMockUtils();
+    private final SlackMockUtils mockUtils = new SlackMockUtils();
 
-    public SlackConfigEntityTest() {
-        super(mockUtils, SlackDistributionConfigEntity.class);
+    @Override
+    public MockUtils<?, ?, SlackDistributionConfigEntity, ?> getMockUtil() {
+        return mockUtils;
```

Comment:
You could just do 'new SlackMockUtils()' in the getMockUtil

Reply:
Will java allow you to change the return type to SlackMockUtils and still be a valid override for the abstract method?